library/src/main/res/values/library_support_annotations_strings.xml

```diff
@@ -8,11 +8,13 @@
     <!-- Library section -->
     <string name="library_support_annotations_libraryName">Support Annotations</string>
     <string name="library_support_annotations_libraryDescription">The Annotation package provides APIs to support adding annotation metadata to your apps.</string>
-    <string name="library_support_annotations_libraryWebsite"></string>
+    <string name="library_support_annotations_libraryWebsite">https://developer.android.com/reference/android/support/annotation/package-summary.html</string>
     <string name="library_support_annotations_libraryVersion">23.2.1</string>
     <!-- OpenSource section -->
     <string name="library_support_annotations_isOpenSource">true</string>
     <string name="library_support_annotations_repositoryLink"></string>
+    <!-- ClassPath for autoDetect section -->
+    <string name="library_support_annotations_classPath">android.support.annotation.MainThread</string>
```

Comment:
It does show up in the project window as 'external library' and gets auto-detected with this classpath.  Is there a reason not to include it here?